src/openms_gui/source/VISUAL/LayerDataBase.cpp

```diff
@@ -204,15 +201,7 @@ namespace OpenMS
     FileTypes::Type type = FileHandler::getType(filename);
     vector<PeptideIdentification> identifications;
     vector<ProteinIdentification> protein_identifications;
-    if (type == FileTypes::MZIDENTML)
-    {
-      MzIdentMLFile().load(filename, protein_identifications, identifications);
-    }
-    else
-    {
-      String document_id;
-      IdXMLFile().load(filename, protein_identifications, identifications, document_id);
-    }
+    FileHandler().loadIdentifications(filename, protein_identifications, identifications, {FileTypes::MZIDENTML, FileTypes::IDXML});
```

Comment:
what about this document id thingy? do we even use that in other code. do you know?

Reply:
I think there was one case across the whole codebase where it was used. Otherwise it was just set and ignored. I retained it where it was used.